app/Http/Controllers/BankRegistryController.php

```diff
@@ -138,10 +138,19 @@ public function destroy(BankRegistry $bankRegistry, Request $request)
     }
 
     public function search(Request $request){
-        $query = $request->input('query');
-        $banks = BankRegistry::where('bank_name', 'like', "%$query%")->get();
+        switch ($request->input('action')){
+            case 'search':
+                $bnquery = $request->input('bn-query');
+                $btquery = $request->input('bt-query');
+                $banks = BankRegistry::where('bank_name', 'like', "%$bnquery%")->where('bank_type_id', $btquery)->get();
+            break;
+
+            case 'clear':
+                $banks = null;
+            break;
+        }
+        return view('home')->with('banks', $banks);
```

Comment:
1. Why did you use home as view of the search? I think you can use the existing index page of bank registry just pass the filtered array of data to view.
2. If no data has been posted it should return all bank registries by default with the status of 0. Always include isInactive status 0 in your query.